PS8lite.Rmd

```diff
@@ -187,6 +189,34 @@ MiscVal + MoSold + YrSold" %>%
 Fit one Random Forest model `model_rf` using any approach you want.
 
 ```{r}
+# Fitting with Caret method:
+# Define cross-validation settings: 10-fold CV
+fit_control <- trainControl(method = "cv", number = 10)
+
+# Setup search grid of "tuning" parameters
+mtry <- 2:5
```

Comment:
Hey @slee26, you're done! But here is what a "code review" would look like. A "code review" is a back and forth between individuals to get the code in the pull request in the desired shape before merging. For example, since we have 36 predictors, it might have been worth setting `mtry <- 2:36`. However, no need for now; you're done